pages/services/edge-lb/data.yml

```diff
@@ -0,0 +1,34 @@
+# Common values:
+
+packageName: edge-lb
+serviceName: edgelb
+techName: Edge-LB
+techShortName: Edge-LB
+
+# Values specific to certain templates:
+
+install:
+  minNodeCount: three
+  nodeDescription: with three Apache Cassandra nodes
```

Comment:
Why do we mention Cassandra here?

Reply:
This file was copied from Cassandra. I have not modified it, since I am not the expert here.